adapter/pkg/oasparser/utills/openApiUtills_test.go

```diff
@@ -63,7 +61,6 @@ func TestFindSwaggerVersion(t *testing.T) {
 			message: "when openAPi version is not provided",
 		},
 	}
```

Comment:
I think its better to keep a newline before starting a conditional section. so shall we keep this
```suggestion
	}

```